src/org/labkey/test/tests/DomainDesignerTest.java

```diff
@@ -0,0 +1,265 @@
+package org.labkey.test.tests;
+
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.labkey.remoteapi.domain.DomainResponse;
+import org.labkey.remoteapi.domain.GetDomainCommand;
+import org.labkey.remoteapi.query.SaveRowsResponse;
+import org.labkey.test.BaseWebDriverTest;
+import org.labkey.test.Locator;
+import org.labkey.test.SortDirection;
+import org.labkey.test.TestTimeoutException;
+import org.labkey.test.categories.DailyA;
+import org.labkey.test.components.DomainDesignerPage;
+import org.labkey.test.components.DomainFieldRow;
+import org.labkey.test.components.DomainFormPanel;
+import org.labkey.test.params.FieldDefinition;
+import org.labkey.test.util.DataRegionTable;
+import org.labkey.test.util.ListHelper;
+import org.labkey.test.util.PortalHelper;
+import org.labkey.test.util.TestDataGenerator;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+@Category({DailyA.class})
+public class DomainDesignerTest extends BaseWebDriverTest
+{
+    @Override
+    protected void doCleanup(boolean afterTest) throws TestTimeoutException
+    {
+        super.doCleanup(afterTest);
+    }
+
+    @BeforeClass
+    public static void setupProject()
+    {
+        DomainDesignerTest init = (DomainDesignerTest) getCurrentTest();
+
+        init.doSetup();
+    }
+
+    private void doSetup()
+    {
+        _containerHelper.createProject(getProjectName(), null);
+    }
+
+    @Before
+    public void preTest() throws Exception
+    {
+        goToProjectHome();
+        navBar().enterPageAdminMode();
+        new PortalHelper(getDriver()).addBodyWebPart("Sample Sets");
+        navBar().exitPageAdminMode();
+    }
+
+    @Test
+    public void testListNumericFormatting() throws Exception
+    {
+        String listName = "NumericFieldsList";
+        FieldDefinition.LookupInfo lookupInfo = new FieldDefinition.LookupInfo(getProjectName(), "lists", listName);
+
+        ListHelper helper = new ListHelper(getDriver());
+        helper.createList(getProjectName(), listName, ListHelper.ListColumnType.Integer,"Key");
+
+        // go to the new domain designer and do some work here
+        DomainDesignerPage domainDesignerPage = DomainDesignerPage.beginAt(this, getProjectName(), "lists", listName);
+
+        DomainFieldRow integerRow = domainDesignerPage
+                .fieldProperties(listName)
+                .addField("integerField")
+                .setType("Integer")
+                .expand()
+                .setNumberFormat("###,###,###")
+                .setScaleType("LINEAR")
+                .setDescription("field for an Integer")
+                .setLabel("IntegerFieldLabel");
+
+        DomainFieldRow decimalRow = domainDesignerPage
+                .fieldProperties(listName)
```

Comment:
There will be cases where we have more than one domainFormPanel on the page.  I'll leave the ability to pick which one in place (even though I don't think it's burdensome to require the caller to instantiate it if they need to reuse it) because not doing that would feel a bit like selling a car with the steering fixed in place because that's the default

Reply:
I'm just talking about in this test, not the design of the panel component.
Just grab the result `domainDesignerPage.fieldProperties` once and use it to create both new fields.

Reply:
Ah, that makes sense.